### DISCUSSION TITLE
Forcings evaluation methods (Precipitation, Temperature, Radiation)

### DISCUSSION THREAD

**Discussion by sfoks** [hytest-org/hytest#381] [Model Evaluation Methods]
## Request
Please feel free to use this space to discuss the evaluation methods of precipitation, temperature, and radiation variables, propose ideas, and post questions.

## About
Currently our [draft forcings evaluation workflow](https://github.com/hytest-org/hytest/tree/main/evaluation/tutorials/CONUS404) examines gridded simulated data ([CONUS404](https://doi.org/10.5066/P9PHPK4F)) vs gridded reference (or observational; [PRISM](https://prism.oregonstate.edu/) and [CERES-EBAF](https://ceres.larc.nasa.gov/data/)) data, and also examines simulated gridded data (CONUS404) versus observational point data ([GHCN](https://www.ncei.noaa.gov/products/land-based-station/global-historical-climatology-network-daily), [USCRN](https://www.ncei.noaa.gov/products/land-based-station/us-climate-reference-network)). 

The workflow calculates the mean, median, and standard deviation annually, and also calculates the bias, percent bias, mean absolute error, root mean squared error, Pearson and Spearman correlation coefficients between the simulated and observational reference data. The example is for the Delaware River Basin and evaluations are performed at a HUC6 spatial unit. 

We are looking to develop this workflow further by incorporating comparisons of these variables over different regions, seasonally, etc. Any suggestions are welcome, and thank you for your time!

**Comment by sfoks** [comment]
One idea that has come up is understanding trends in modeled data, and therefore performing trend analysis in forcing variables.

**Reply by mirizarry-ortiz** [reply]
I am working on just that for FL but just for precip. for now. May look at temperature soon.
